crates/astria-composer/src/executor/mod.rs

```diff
@@ -192,6 +196,10 @@ impl Executor {
     /// An error is returned if connecting to the sequencer fails.
     #[instrument(skip_all, fields(address = %self.address))]
     pub(super) async fn run_until_stopped(mut self) -> eyre::Result<()> {
+        let _chain_id_result = self
```

Comment:
I don't believe this needs to be assigned to anything as written, but bigger note down on mild update which can be made

Reply:
Resolved in b8d148f.